data/human/hails.txt

```diff
@@ -2962,6 +2996,34 @@ phrase "hostile disabled"
 		"."
 		"!"
 
+# Shakespearean insults.
+phrase "disabled hostile"
+	word
+		"You piece of"
+		"Son of a"
```

Comment:
I feel that "Son of a" doesn't work for some the shakespearean insults, for example "Son of a bagage" or "Son of a wagtail" sounds a bit weird.